backend/tests/modules/publishing/test_router.py

```
@@ -1 +1,705 @@
+from app.dependencies import get_current_user, get_current_user_optional
+from modules.publishing.router import router, _build_simulation_response
+from modules.publishing.service import PublishingService
+from common.db.models import Simulation, SimulationPersona, SimulationScene, User
+
+
+@pytest.fixture
+def mock_user(db_session: Session):
+    """Create a test user in the database."""
+    user = User(
+        email="test@example.com",
+        hashed_password="hashed",
```

Ruff (0.14.8)

Possible hardcoded password assigned to argument: "hashed_password"

(S106)